app/assets/javascripts/app.coffee

```diff
@@ -0,0 +1,9 @@
+window.App ||= {}
+
+App.init = ->
+  $('[data-provider="summernote"]').each ->
+    $(this).summernote height: '400px'
+    return
+
+$(document).on "turbolinks:load", ->
+  App.init()
```

Comment:
I would make it a little different and add `[data-provider="summernote"]` at `init` method. like so: 

`App.init('[data-provider="sumernote"]')`

also I think summernote should be its own component? so you would just initialize in the `#init` method?
```
App.init = ->
  SummerNote.init('[data-provider="summernote"]', '400px')
 ....
```

what do you think about it ?